planner/core/common_plans.go

```diff
@@ -350,10 +350,6 @@ func (e *Execute) getPhysicalPlan(ctx context.Context, sctx sessionctx.Context,
 	e.Plan = p
 	_, isTableDual := p.(*PhysicalTableDual)
 	if !isTableDual && prepared.UseCache {
-		err = e.setFoundInPlanCache(sctx, true)
```

Comment:
It should have been removed by https://github.com/pingcap/tidb/pull/16594/files, wonder why it appears again.